src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +797,105 @@ def register_deploy_wrapper(*args, **kwargs):
+                data["Instance Type"].append(
+                    deployment_config.get("DeploymentConfig").get("InstanceType")
+                )
+                data["Selected"].append(
+                    "Yes" if self.config_name == deployment_config.get("ConfigName") else "No"
+                )
+
+                if i == 0:
+                    for benchmark_metric in benchmark_metrics:
+                        column_name = (
+                            f"{benchmark_metric.get('name')} ({benchmark_metric.get('unit')})"
+                        )
+                        data[column_name] = []
+
+                for benchmark_metric in benchmark_metrics:
+                    column_name = f"{benchmark_metric.get('name')} ({benchmark_metric.get('unit')})"
+                    if column_name in data.keys():
+                        data[column_name].append(benchmark_metric.get("value"))
+
+        df = pd.DataFrame(data)
+        headers = {"selector": "th", "props": "text-align: left;"}
+        df.style.set_caption("Benchmark Metrics").set_table_styles([headers]).set_properties(
+            **{"text-align": "left"}
+        )
+        return df
+
+    def list_deployment_configs(self) -> List[Dict[str, Any]]:
+        """List deployment configs for ``This`` model in the current region.
+
+        Returns:
+            A list of deployment configs (List[Dict[str, Any]]).
+        """
+        if self._metadata_configs is None:
+            self._metadata_configs = get_jumpstart_configs(
+                region=self.region,
+                model_id=self.model_id,
+                model_version=self.model_version,
+                sagemaker_session=self.sagemaker_session,
+            )
+
+            self._deployment_configs = [
+                self._convert_to_deployment_config_metadata(config_name, config)
+                for config_name, config in self._metadata_configs.items()
+            ]
+
+        return self._deployment_configs
+
+    def _convert_to_deployment_config_metadata(
+        self, config_name: str, metadata_config: JumpStartMetadataConfig
+    ) -> Dict[str, Any]:
+        """Retrieve deployment config for config name.
+        Args:
+            config_name (str): Name of deployment config.
+            metadata_config (JumpStartMetadataConfig): Metadata config for deployment config.
+        Returns:
+            A deployment metadata config for config name (dict[str, Any]).
+        """
+        default_inference_instance_type = metadata_config.resolved_config.get(
+            "default_inference_instance_type"
+        )
+
+        instance_rate = get_instance_rate_per_hour(
+            instance_type=default_inference_instance_type, region=self.region
+        )
+
+        benchmark_metrics = metadata_config.benchmark_metrics.get(default_inference_instance_type)
+        if benchmark_metrics is not None and instance_rate is not None:
```

Comment:
If there's failure on fetching instance rate, we should still show the metric numbers.